### DISCUSSION TITLE
Charging of battery is added to house consumption

### DISCUSSION THREAD

**Discussion by perjmolsen** [ulic75/power-flow-card#163] [Q&A]
HI

When my solar battery is charging, my house consumption is also rising. 
Can it be fixed so that my house consumption only shows the actual house consumption?

When charging of my battery is at 100% my house consumption is back to normal.
I am using split entries, see below:
type: custom:power-flow-card
watt_threshold: 1
kw_decimals: 2
w_decimals: 1
min_flow_rate: 0.75
max_flow_rate: 6
entities:
  battery:
    consumption: sensor.battery_discharging_w
    production: sensor.battery_charging_w
  battery_charge: sensor.omcuch9089_battery_charge_2
  grid:
    consumption: sensor.power_consumed
    production: sensor.power_produced
  solar: sensor.ppv_watt_combined

![solar snip](https://user-images.githubusercontent.com/126662740/234287773-e00bce5b-8a42-49af-a51c-1b14cfe82ca2.jpg)

**Comment by ulic75** [comment]
The logic works as follows using the above data.

sesnor.ppv_watt_combined reports 2.62 is being produced from solar
sensor.power_produced reports that 0.08 is being sent/returned to the grid
sensor.batter.charging_w reports that 0.58 is being sent to the battery

Therefore the assumption is that everything else (home) is consuming 1.96.

2.62 - .08 - .58 = 1.96

It may be that other things are consuming power from solar, but everything get's bundled up into Home.

**Reply by titizb33** [reply]
![powerflow](https://github.com/ulic75/power-flow-card/assets/77579632/07650fc7-714b-4b79-b98c-7b6752dff4d8)

Hi,
Good morning,
At home the battery charge is not deducted from the consumption of my house.
I don't understand the behavior of the battery bubbl


1100 - 14 - 249 = 837 <> 1100



type: custom:power-flow-card-plus
entities:
  battery:
    entity: sensor.shellyplus1pm_90380c37bd4c_switch_0_power
    state_of_charge: ''
    invert_state: true
    icon: mdi:storage-tank
    name: ECS
    display_state: one_way
  grid:
    entity: sensor.shellyem_b9ed49_channel_1_power
    display_state: one_way
  solar:
    entity: sensor.est_power
  individual1:
    entity: sensor.shellyem_b9ed49_channel_2_power
    name: Clim
    icon: mdi:air-conditioner
  individual2:
    entity: sensor.piscine_power
    name: Piscine
    icon: mdi:pool
  home: {}
clickable_entities: true
display_zero_lines: false
use_new_flow_rate_model: true
w_decimals: 0
kw_decimals: 1
min_flow_rate: 0.75
max_flow_rate: 6
max_expected_power: 2000
min_expected_power: 0.01
watt_threshold: 1000

**Reply by ulic75** [reply]
You'll probably want to repost this in the power-flow-card-plus repository discussions.  The card you are using is not this card.
